calebwagner/sink-repair
# added requests module and formatted css

I created a requests function that will list out order descriptions in an unordered list. Also, I formatted the web page with css.

Steps to check code:
1. git fetch --all 
2. git checkout cjw-requests.js-module
3. code .
4. serve
5. check Requests.js module for requests function
6. check in the browser for the description generated by the function in the browser
